### PR TITLE
gh-93453: Make get_event_loop() an alias of get_running_loop()

### PR DESCRIPTION
**Pull request by serhiy-storchaka**
#93453

### CODE REVIEW

File: Lib/asyncio/events.py [hunk 1]
```diff
@@ -823,22 +803,20 @@ def set_child_watcher(watcher):
...
     pass
 else:
     # Alias C implementations for testing purposes.
     _c__get_running_loop = _get_running_loop
     _c__set_running_loop = _set_running_loop
     _c_get_running_loop = get_running_loop
-    _c_get_event_loop = get_event_loop
-    _c__get_event_loop = _get_event_loop
+
+get_event_loop = get_running_loop
```
Comment (gvanrossum) [inline Lib/asyncio/events.py:819]: Isn't this a duplicate of line 778?

**Review by gvanrossum — COMMENTED** [review]
I think we should eventually do this, but I'm not keen to do it right now. I think we should first deprecate policies, child watchers, and set_event_loop(). See GH-94597.

**Comment by gvanrossum** [comment]
I think we've changed our mind on this, and we're not doing this, ever, right? So then it should be closed.

**Comment by serhiy-storchaka** [comment]
I thought that I closed this PR. Thank you for a reminder.

**Pull request closed without merging**